### ISSUE TITLE
New Snippet Title requires first character to be typed twice

### ISSUE THREAD

**Issue by ktalley-vtech** [antonreshetov/massCode#85]
On massCode for MacOS (version 1.1.0, but experienced same issue on earlier versions), whenever I add a new snippet (i.e. click "+"), I see the highlighted/selected text "Untitled snippet" in the right-hand panel (up in the title area) -- so far, so good.  But when I start typing, the first character typed remains highlighted/selected, so that when I type the next (second) character, it overwrites the first character (and THEN the highlighting correctly goes away for the title text).  So basically, I have to type the first character of a snippet title TWICE for it to work correctly.  Not a show-stopper, but certainly an irritation.  :)

Love massCode -- great job!

Regards,
Kevin T.
macOS Catalina - 10.15.2 (massCode v1.1.0)

**Comment by estarq** [comment]
Not only Mac - Linux too.

```
Version: 1.1.0
Electron: 7.1.6
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279-23-electron.0
OS: Linux x64 5.3.0-42-generic
```

**Comment by Loigzorn** [comment]
Seems to be a issue on all platforms - Windows too.

>            Version: 1.1.0
              Electron: 7.1.6
              Chrome: 78.0.3904.130
              Node.js: 12.8.1
              V8: 7.8.279.23-electron.0
              OS: Windows_NT x64 10.0.18362
>

**Comment by alexrollin** [comment]
metoo

**Comment by antonreshetov** [comment]
Fix will be added to the new release

**Comment by antonreshetov** [comment]
Closed because it was fixed in #87

**Comment by ktalley-vtech** [comment]
Thanks for fixing this Anton!  I've been adding snippets this morning and I have a big smile on my face!  :)
